eng/Version.Details.xml

```diff
@@ -302,6 +302,10 @@
       <Uri>https://github.com/dotnet/runtime</Uri>
       <Sha>977a00fb9d587a6c1292fb3af3992038ddcd3016</Sha>
     </Dependency>
+    <Dependency Name="Microsoft.NETCore.BrowserDebugHost.Transport" Version="6.0.0-dev">
+      <Uri>https://github.com/dotnet/runtime</Uri>
+      <Sha>6cc7cffaff2e0413ee84d9a7736f9ae1aa9a3f12</Sha>
+    </Dependency>
```

Comment:
Is this a local change that bled into your PR❔ Version should match something actually coming out of the dotnet/runtime.

Reply:
We're still working on getting a working package out of the runtime repo. There's been quite a few hiccups so I ended up using a dev build to unblock myself.

I marked this PR as ready for review so that the framework-side changes can be reviewed. I won't be merging it until we have a working package and will update the versions accordingly then.